account_move_journal_template/models/account_move.py

```diff
@@ -0,0 +1,116 @@
+# Copyright 2022 Therp BV <https://therp.nl>
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl.html).
+from odoo import _, fields, models
+
+
+class AccountMove(models.Model):
+    _inherit = "account.move"
+
+    journal_entry_ids = fields.One2many(
+        comodel_name="account.move", inverse_name="journal_entry_id",
+    )
+    journal_entry_id = fields.Many2one(comodel_name="account.move")
```

Comment:
And again if I understand correctly this is the link FROM the generated extra move TO the original invoice. Please add a help= also here.

Reply:
And maybe rename field to invoice_move_id?